acceptance-tests/src/main/kotlin/org/ostelco/at/TestSuite.kt

```diff
@@ -49,6 +51,41 @@ class TestSuite {
         }
     }
 
+    /**
+     * Test of recurring payments requires that:
+     *    1) The NGROK service docker is running (Docker instance).
+     *    2) The NGROK_AUTH_TOKEN env. variable set to the NGROK token.
+     *    2) Webhooks enabled and configured to use the NGROK endpoint on Stripe.
+     * As NGROK only allows for up to 40 connections per minute, which with Stripe
+     * events means max 40 events per minute, the recurring payment tests has to
+     * be run seperately from the other tests. Otherwise the limit will be exceeded
+     * and tests will fail.
+     * Note that if the NGROK_AUT_TOKEN variable is not set, then the test will be
+     * skipped altogether.
+     */
+    @Test
+    @EnabledIfEnvironmentVariable(named = "NGROK_AUTH_TOKEN", matches="\\S+")
```

Comment:
Do you want these tests to be run on CI?  Should we add this ENV var to CI setup?

Reply:
Yes, but we will first have to create an account for CI with ngrok.io first. And maybe consider a baslc or pro subscription? Being able to use "custom subdomain" would simplify the setup a bit...